components/app/Notification.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+import { useNotificationStore } from '~/stores/nofiStore';
+import { onMounted, onUnmounted, defineAsyncComponent } from 'vue';
+const props = withDefaults(
+    defineProps<{
+        id: string
+        message: string
+        type?: "success" | "error"
+    }>(),
+    {
+        type: 'success',
+    }
+)
+const variant = computed(() => {
+    return `-${props.type}`
+})
+const notificationStore = useNotificationStore()
+const handleRemoveNoti = () => {
+    notificationStore.removeNotification(props.id)
+}
+
+
+onMounted(() => {
+    setTimeout(() => {
```

Comment:
const timerId = ref()
```suggestion
    timerId.value = setTimeout(() => {
```